### DISCUSSION TITLE
Pods Rebalancing

### DISCUSSION THREAD

**Discussion by WilliamWeberBerrutti** [OpenHV/OpenHV#879] [Ideas]
## Common to all factions

Machinegun Pods
Missile Pods (make it be the electric one for Yuruki to fit the faction's thematic better)
Technicians
Elite Pods
Broker Pods (let both factions build Ore Smelters?)

## Synapol only

Mortar Pods (change it to be Synapol's sniper, having faster moving projectiles and the same range maybe)
Flamer Pod

## Yuruki only

Sniper Pods
Bomber Pods

**Comment by Mailaender** [comment]
Broker Pods are the counterparts to Ore Smelters so you can't make them common.

**Reply by WilliamWeberBerrutti** [reply]
The question is: if you make Broker Pods, that means you won't be able to build units from Barracks, making your army composition weaker. Since it's easier for the Main Base to be idle than Barracks, Ore Smelters would be more advantageous to build than Broker Pods (even if it costs power and can't move), that's why I thought on making both Ore Smelters and Broker Pods available for both factions, so depending of the situation, the player have an option to build one or both of them (but this situation may be too much. Maybe it encourages aggressive play so players can't turtle by building both simultaneously?).

**Reply by WilliamWeberBerrutti** [reply]
Or, what if Broker Pod is available for both factions only after capturing the Resource Extractor?

**Reply by Mailaender** [reply]
The different style in late game economy is an attempt to make both factions less similar, which was a common criticism.

**Comment by WilliamWeberBerrutti** [comment]
Also, what if you could build the Main Base on the water?
